conf/modules.config

```diff
@@ -275,9 +275,13 @@ process {
         ]
     }
 
+    // squash-ploidy is necessary to be able to match het-hom changes
     withName: "RTGTOOLS_VCFEVAL" {
         ext.prefix = {"${meta.id}.${params.truth_id}.${meta.caller}"}
-        ext.args   = {"--all-record"}
+        ext.args   = {["--all-record ",
+                    (params.analysis == somatic)      ? '--squash-ploidy' : ''
+                    ].join('').trim()
```

Comment:
I think the trim might be unnecessary because you do join('') before. Not a big thing though.